shapeshift/web
# feat: add FOX farming APR hook

## Description

This adds a hook to fetch the farming APR from ETH/FOX UniV3.

## Notice




## Pull Request Type


## Issue (if applicable)

Partially tackles #1834


## Risk

None since this isn't added as app code yet.


## Testing

- Consume useFarmingApr from e.g src/plugins/foxPage/components/Layout.tsx
- Verify that loading state is working
- Verify that the farming APR is matching the one in https://fox.shapeshift.com/fox-farming


## Screenshots (if applicable)

<img width="441" alt="image" src="https://user-images.githubusercontent.com/17035424/171680917-e4ec8fae-75aa-498c-964e-6b1bbb983e25.png">
